### PR TITLE
feat(contrib/coreos): add preseed script for Deis components

### PR DESCRIPTION
**Pull request by carmstrong**
Also adds a new documentation section for production-ready recommendations.

### CODE REVIEW

File: docs/managing_deis/production_deployments.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+can consider enabling a firewall on the CoreOS hosts as well as the router component.
+
+See :ref:`security_considerations` for details.
+
+Back up data
+------------
+
+Backing up data regularly is recommended. See :ref:`backing_up_data` for steps.
+
+Configuring logging and monitoring
```
Comment (bacongobbler) [inline docs/managing_deis/production_deployments.rst]: Staying consistent with the rest of the titles: "Configure logging and monitoring"

Comment (carmstrong) [inline docs/managing_deis/production_deployments.rst]: Good catch. Fixed.

**Comment by mboersma** [comment]
These docs and the `preseed` script are great. Code LGTM.

**Comment by bacongobbler** [comment]
If we moved `deisctl` to swarm, this'd be as easy as a `docker pull` from a swarm manager. code LGTM

**Pull request merged**